src/main/webapp/resources/js/components/ant.design/table-renderers.js

```diff
@@ -26,14 +26,16 @@ export const nameColumnFormat = ({ url }) => {
           {name}
         </a>
       );
-    }
+    },
   };
 };
 
 export const dateColumnFormat = ({ className = "" } = {}) => ({
   sorter: true,
   width: 230,
-  render: date => (
-    <span className={className}>{formatInternationalizedDateTime(date)}</span>
-  )
+  render: (date) => (
+    <span className={className}>
+      {date ? formatInternationalizedDateTime(date) : ""}
```

Comment:
Comment on here about what it's here for please.

Reply:
Updated in edb24b1ec16b6bded0bb70798bd75158094412be